docs/content/webhooks-app.md

```diff
@@ -92,27 +98,125 @@ Below are examples showcasing various display formats your webhooks will adopt w
 
 ![title only post](./images/hook-all.png)
 
+The example below demonstrates the simplest data structure you would `POST` to achieve the above result.
+
+```json
+{
+  "items": [
+    {
+      "author": {
+        "displayName": "David Jones",
+        "profileImage": {
+          "url": "https://example.com/profiles/davidjones.jpg"
+        }
+      },
+      "content": {
+        "title": {
+          "content": "Welcome to our new London Office!"
+        },
+        "body": {
+          "content": "We are excited to announce the opening of our new office in London. The new office is located in the heart of the city and will be the new home for our growing team. We ar elooking forward to welcoming our clients and partners to our new office."
+        }
+      },
+      "attachments": [
+        {
+          "contentType": "image",
+          "url": "https://example.com/images/sample.jpg"
+        }
+      ]
+    }
+  ]
+}
+```
+
 </details>
 
 <details>
   <summary>Show title only screenshot</summary>
 
 ![title only post](./images/hook-title-only.png)
 
+The example below demonstrates the simplest data structure you would `POST` to achieve the above result.
+
+```json
+{
+  "items": [
+    {
+      "author": {
```

Comment:
For title only do we remove the author too for demo purposes?

Reply:
The author is in all the images, so to be consistent with the image I kept the author there. Author is in all the images in fact.

Reply:
Sorry that's what I mean maybe we should create images without the author to show the real bare minimum for the webhook item. It doesn't need an author but might read to the user they do in this case?

Reply:
Yep, I agree. I have altered the images now and updated the content. The author is now only on the full content one.